### DISCUSSION TITLE
Recursos utiles para la aventura explorers

### DISCUSSION THREAD

**Discussion by AlvaroFierro** [Launch-X-Latam/MisionFrontEnd#272] [General]
Explorers, a lo largo de unos meses he ido recabando algunas herramientas utiles para cuando uno esta escribiendo codigo, les dejo los links y una pequeña descripcion

Espero les sirva explorers 🙌

## Coolors - The super fast color schemes generator!

Es un generador de paletas de colores 
[coolors.co](https://coolors.co/)

## mydevice.io : web devices capabilities

Te ayuda a ver los pixeles de algunas pantallas de los dispositivos, ideal para los breakpoints
[mydevice.io](https://www.mydevice.io/#compare-devices/#compare-devices)

## Flexbox Froggy - A game for learning CSS flexbox
Es un juego que te permite aprender flexbox de manera intuitiva
[flexboxfroggy.com](https://flexboxfroggy.com/)

## A Complete Guide to Flexbox | CSS-Tricks
Una guia para los estilos de flexbox
[css-tricks.com](https://css-tricks.com/snippets/css/a-guide-to-flexbox/snippets/css/a-guide-to-flexbox/)

## FLEX: A simple visual cheatsheet for flexbox
Una ayuda para flex
[flexbox.malven.co](https://flexbox.malven.co/)

## CSS Flexbox: A simple visual cheatsheet for flexbox
Una ayuda para flex, se parece a la anterior
[flexboxsheet.com](https://flexboxsheet.com/)

# flexboxsheet9
Un PDF que te ayuda con los contenedores de flex
[jonitrythall.com](https://jonitrythall.com/content/images/flexboxsheet.pdf
)

## Color Palettes for Designers and Artists - Color Hunt
Paletas de colores para tus diseños 
[colorhunt.co](https://colorhunt.co/)

**Comment by LauraHenao27** [comment]
Vine por cobre y encontré oro, Muchas gracias:)

**Comment by Amhedriel** [comment]
# **Buenas noches colegas**

Agrego unas cuantas páginas de recursos gratuitos para el diseño.

## [flaticon](https://www.flaticon.es/)

### Accede a +8.4M iconos vectoriales y stickers
Descarga iconos y stickers gratis para tus proyectos. Recursos hechos por y para diseñadores. Formatos PNG, SVG, EPS, PSD y CSS.

---

## [unDraw](https://undraw.co/illustrations)

### Imágenes vectoriales

Busque las imágenes que se ajusten a sus necesidades y haga clic para descargar. Utilice la generación de imágenes en color sobre la marcha para que coincida con su identidad de marca.

---

## [boxicons](https://boxicons.com/)

### High Quality Web Icons

Iconos (nunca son suficientes) simples de código abierto cuidadosamente diseñados para diseñadores y desarrolladores.

---

## [atomicons](https://atomicons.com/)

### Iconos hermosos y altamente personalizables

256Iconos gratuitos que puedes personalizar para tu proyecto.

---
---

Espero les sean útiles.

Saludos.

**Reply by AlvaroFierro** [reply]
Gracias @Amhedriel

**Comment by David-Carr-C** [comment]
Excelente, como guardo este hilo de github?

**Reply by Amhedriel** [reply]
Arriba, en la barra azul, debajo de tu foto de perfil tendría que mostrarte la opción de ***SAVE***  al lado de **Mark as unread**.

Saludos
